.github/workflows/ci-cd.yml

```diff
@@ -91,7 +103,13 @@ jobs:
         run: |
           python -m venv venv
           source venv/bin/activate
-          pip install grpcio-tools==1.59.2 grpclib==0.4.7
+          if [ "${{ matrix.python-version }}" == "3.9" ]; then
```

Comment:
This check is just meant to assert that we haven't added any third-party dependencies to `python -m modal._container_entrypoint` that aren't in the expected container requirements, so the specific way that we compile the protos isn't super important because whatever choice we make wouldn't introduce transitive dependencies. But (as we're discussing on Slack) there definitely are larger and trickier questions about our proto packaging!